### PR TITLE
CDD-1411_goods_arrival

### PR DESCRIPTION
**Pull request by paulblakehmrc**

### CODE REVIEW

File: test/acceptance/ImportsControllerSpec.scala [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+    withHeader(X_FORWARDED_HOST, equalTo("MDTP")).
+    withHeader(AUTHORIZATION, equalTo(s"Bearer ${ExternalServicesConfig.AuthToken}"))
+
+  override def fakeApplication(): Application  = new GuiceApplicationBuilder().configure(Map(
+    "microservice.services.validatemovement.host" -> ExternalServicesConfig.Host,
+    "microservice.services.validatemovement.port" -> ExternalServicesConfig.Port,
+    "microservice.services.validatemovement.context" -> importMovementUrl,
+    "microservice.services.validatemovement.bearer-token" -> ExternalServicesConfig.AuthToken,
+    "microservice.services.goodsarrival.host" -> ExternalServicesConfig.Host,
+    "microservice.services.goodsarrival.port" -> ExternalServicesConfig.Port,
```
Comment (googley42) [inline test/acceptance/ImportsControllerSpec.scala]: same URL for 2 different message type does not look right. Also name `importMovementUrl` needs to be message specific.

File: test/acceptance/ImportsControllerSpec.scala [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+  }
+
+  override protected def beforeEach() {
+    resetMockServer()
+  }
+
+  override protected def afterAll() {
+    stopMockServer()
+  }
+
```
[the remark this answers is not in the document]
Comment (paulblakehmrc) [reply test/acceptance/ImportsControllerSpec.scala]: Done

File: test/acceptance/ImportsControllerSpec.scala (hunk 1 above)
Comment (paulblakehmrc) [reply test/acceptance/ImportsControllerSpec.scala]: Fixed

File: app/uk/gov/hmrc/customs/inventorylinking/imports/connectors/OutgoingRequestBuilder.scala [hunk 3]
```diff
@@ -19,8 +19,8 @@ package uk.gov.hmrc.customs.inventorylinking.imports.connectors
 import javax.inject.{Inject, Singleton}
 
 import uk.gov.hmrc.customs.api.common.config.ServiceConfig
+import uk.gov.hmrc.customs.inventorylinking.imports.model.HeaderNames.{XBadgeIdentifier, XClientId}
 import uk.gov.hmrc.customs.inventorylinking.imports.model.RequestInfo
-import uk.gov.hmrc.customs.inventorylinking.imports.controllers.HeaderNames.{XBadgeIdentifier, XClientId}
 import uk.gov.hmrc.customs.inventorylinking.imports.xml.PayloadDecorator
 
 import scala.xml.NodeSeq
```
[the remark this answers is not in the document]
Comment (paulblakehmrc) [reply app/uk/gov/hmrc/customs/inventorylinking/imports/connectors/OutgoingRequestBuilder.scala]: Done

**Comment by googley42** [comment]
:+1:

**Pull request merged**